lib/airbrake-ruby/code_hunk.rb

```diff
@@ -44,5 +45,11 @@ def get(file, line)
       return { 1 => '' } if lines.empty?
       lines
     end
+
+    private
+
+    def get_from_cache(file)
+      @cache[file] ||= File.foreach(file)
```

Comment:
It is probably fine as is, but I was thinking about caching the result (code hunk given file&line) instead of whole file which can easily be 100kb * 100 cache entries = 10 mbs. I don't insist though.

Reply:
We can always lower the limit of cache. 100 unique files per backtrace is very unrealistic (even 100 frames is hard to achieve).